### ISSUE TITLE
Error in parsing -U option

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#309]
**Description of the problem**

**Full script that generated the error**

```
gmt basemap -R0/10/0/5 -JX10c/3c -Bx1 -By1 -U"logo"+jBL+o-1.5c/-1.5c -png GMT_-U
```

**Full error message**

```
psbasemap [ERROR]: Syntax error -U option. Correct syntax:
	-U[<label>][+c][+j<just>][+o<dx>/<dy>]
	   Plot the time stamp and optional command line or text.
psbasemap [ERROR]: Offending option -Ulo+o+jBL+o-1.5c/-1.5c
```

**System information**

* Operating system: macOS
* Version of GMT: 6.0.0_7b301d7

**Comment by PaulWessel** [comment]
Confirmed, fix is being tested.

**Comment by seisman** [comment]
Fixed in #311.
